models/stg_google_ads.sql

```diff
@@ -11,7 +11,7 @@ with base as (
         'Google Ads' as platform,
         cast(date_day as date) as date_day,
         account_name,
-        cast(external_customer_id as {{ dbt_utils.type_string() }}) as account_id,
+        {% if var('api_source','adwords') == 'google_ads' %} account_id {% else %} external_customer_id as account_id {% endif %} ,
```

Comment:
I'm not too familiar with variables, are they always lowercased? There's not going to be a chance of it being GOOGLE_ADS or something?

Reply:
Good call out @fivetran-reneeli! The variable is set by the user so they very well could in fact set the result to `GOOGLE_ADS`. However, we explicitly tell them that they should use `google_ads` in the [README](https://github.com/fivetran/dbt_ad_reporting/tree/test/google-ads-api#google-ads-api-configuration) so I believe it is safe to strictly look just for the lowercase variable result.

Reply:
If we are using a variable however where we do not explicitly tell the user what to input (if they want to add passthrough columns) it is always a good idea to standardize the results so it can be caught if the users changes the casing.

Reply:
Yeah I agree we should have added a `| lower` filter. However, this filter is not present in the other references of this variable within google_ads (which has already been released) for consistency I will just make a point to call out that this is case sensitive for now.

Thanks team!!